pkg/asset/manifests/ingress.go

```diff
@@ -8,13 +8,17 @@ import (
 	"github.com/pkg/errors"
 
 	configv1 "github.com/openshift/api/config/v1"
+	operatorv1 "github.com/openshift/api/operator/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
 	"github.com/openshift/installer/pkg/asset"
 	"github.com/openshift/installer/pkg/asset/installconfig"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"github.com/openshift/installer/pkg/types"
 )
 
 var (
-	ingCfgFilename = filepath.Join(manifestDir, "cluster-ingress-02-config.yml")
+	clusterIngressConfigFile = filepath.Join(manifestDir, "ingress-cluster-config.yml")
+	defaultIngressControllerFile = filepath.Join(manifestDir, "ingress-default-ingresscontroller.yml")
```

Comment:
I'm not a fan of either of these names — are there any conventions I should follow?

Reply:
a) try keeping new files as yaml
b) the naming has no conventions, something to keep in mind is logically similar files ending up close to each other when looking at directory is easy for users when modifying these.
c) the only time the names matter is, if you want some ordering when pushing this manifest to the cluster, because we push in _almost_ lexicographical increasing order of filenames.